youtube_dl/extractor/podchaser.py

```diff
@@ -0,0 +1,98 @@
+# coding: utf-8
+from __future__ import unicode_literals
+import re
+from datetime import datetime
+from ..utils import float_or_none, try_get, str_to_int
+from ..compat import compat_str
+from .common import InfoExtractor
+
+
+class PodchaserIE(InfoExtractor):
+    _VALID_URL = r'''(?x)
+        https?://(?:www\.)?podchaser\.com/
+        (?:(?:podcasts)|(?:creators))
+        /[\w-]+-
+        (?:
+            (?P<creator_id>[\d]+[\w]+)|(?P<podcast_id>[\d]+))
+        (?:/episodes/[\w\-]+-
+            (?P<id>[\d]+))?'''
+
+    _TESTS = [{
+        'url': 'https://www.podchaser.com/podcasts/cum-town-36924/episodes/ep-285-freeze-me-off-104365585',
+        'info_dict': {
+            'id': '104365585',
+            'title': "Ep. 285 – freeze me off",
+            'description': 'cam ahn',
+            'thumbnail': r're:^https?://.*\.jpg$',
+            'ext': 'mp3',
+            'categories': ['Comedy'],
+            'series': 'Cum Town',
+            'duration': 3708,
+            'timestamp': 1636560059,
+            'upload_date': '20211110'
+        }
+    }, {
+        'url': 'https://www.podchaser.com/podcasts/the-bone-zone-28853',
+        'info_dict': {
+            'id': '28853',
+            'title': 'The Bone Zone',
+            'description': 'md5:c39acd897170a8bf3ad94fc45dc25060',
+        },
+        'playlist_count': 6
+    }, {
+        'url': 'https://www.podchaser.com/creators/todd-glass-107ZzkFiEQ',
+        'info_dict': {
+            'id': '107ZzkFiEQ',
+            'title': 'Todd Glass',
+            'description': 'md5:f0302588c98d474d4e8551f4914042c9',
+        },
+        'playlist_mincount': 48
+    }, {
+        'url': 'https://www.podchaser.com/podcasts/sean-carrolls-mindscape-scienc-699349/episodes',
+        'info_dict': {
+            'id': '699349',
+            'title': "Sean Carroll's Mindscape: Science, Society, Philosophy, Culture, Arts, and Ideas",
+            'description': 'md5:a1ed5db77378a736b3a85a2331cbc05a'
+        },
+        'playlist_count': 25
+    }]
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        audio_id, podcast_id, creator_id = mobj.group('id'), \
+            mobj.group('podcast_id'), mobj.group('creator_id')
+        webpage = self._download_webpage(url, audio_id)
```

Comment:
In testing (py2/3), I got 403 Forbidden, depending on the UA used.

I believe that Cloudflare has broken the web by randomly forbidding certain UA strings (with no diagnostic). If the UA string only includes a 2-component version

Reply:
On my end, this didn't solve the problem. I tried your solution, and I also tried removing the Chrome portion of the UA header altogether. Either way I got a 403 error unless I was using Python 3.7+

Reply:
Totally weird. My test versions are Py 2.7.17 and 3.5.2.

Do you ever get 403 in the browser? If not, what happens with the UA that your browser uses?

Reply:
With Py 2.7.17 on my laptop and Py 2.7.18 on desktop, I get the 403 error, and I tried using my browser's UA. Never gotten 403 in browser.

I don't know if this is relevant, but when I tried running a curl request with only the UA header,
`curl 'https://www.podchaser.com/podcasts/world-record-podcast-with-bren-1020594/episodes/episode-57-geno-randazzo-82092499' -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:94.0) Gecko/20100101 Firefox/94.0'`
it worked on my laptop (which I haven't been using for this project), but on my main PC, the response is always the Cloudflare captcha page.

Reply:
Well, that seems to confirm the responsible party.

In the linked issue it appeared that the **order** of headers might be important to Cloudflare. I suppose that could be different between the various web clients that you tested. It is [observed in the linked issue that the Python web client before 3.7 stores the headers in a `dict`](https://github.com/ytdl-org/youtube-dl/issues/29715#issuecomment-890611459) where the order can't be guaranteed.

Reply:
Ok, then I think I'll try changing this so it downloads the JSON from their API. Seems like there's no problem getting data that way when using Py 2.7.

Reply:
Done 6ddf44e